### DISCUSSION TITLE
Sleep timer

### DISCUSSION THREAD

**Discussion by shreyasskdev** [sayaka-sh/spmp#354] [Q&A]
I need a sleep timer for this app. I search the whole app and I could only find a timer that turns off the song after one song. Doesn't it have a timer like I mentioned or that I couldn't find it? I am currently using Spotify and this is the only feature that holds me back from using this music app. I used other youtube music opensource alternative but none of them is good as this app.

**Comment by sayaka-sh** [comment]
Sorry, this isn't something I'm planning to implement.
